umap/admin.py

```diff
@@ -11,10 +37,22 @@ class TileLayerAdmin(admin.ModelAdmin):
     list_editable = ("rank",)
 
 
-class MapAdmin(admin.GISModelAdmin):
+class MapAdmin(CSVExportMixin, admin.GISModelAdmin):
     search_fields = ("name",)
     autocomplete_fields = ("owner", "editors")
     list_filter = ("share_status",)
+    csv_fields = (
```

Comment:
Can you add the number of features the map contains @yohanboniface  ?

Reply:
Nope, too costly to compute :/